extensions/amp-story/1.0/amp-story.js

```diff
@@ -444,6 +444,7 @@ export class AmpStory extends AMP.BaseElement {
 
         if (args) {
           this.storeService_.dispatch(
+<<<<<<< HEAD
```

Comment:
Looks like there are still change markers here

Reply:
Hmm, I think that I already got rid of this in [this commit](https://github.com/ampproject/amphtml/pull/20944/commits/93d0a06f152dc165cee6cff19a526167de29af69). 

I double checked those lines in the `Files Changed` tab in this PR as well.